sdk/datafactory/azure-resourcemanager-datafactory/src/samples/java/com/azure/resourcemanager/datafactory/ActivityRunsQueryByPipelineRunSamples.java

```diff
@@ -10,6 +10,11 @@
 
 /** Samples for ActivityRuns QueryByPipelineRun. */
 public final class ActivityRunsQueryByPipelineRunSamples {
+    /*
+     * operationId: ActivityRuns_QueryByPipelineRun
+     * api-version: 2018-06-01
+     * x-ms-examples: ActivityRuns_QueryByPipelineRun
+     */
```

Comment:
Is it intended? Two doc blocks

Reply:
This is not a doc block. It is metadata for find the example JSON in swagger.